src/component/axis/CartesianAxisView.ts

```diff
@@ -135,6 +139,37 @@ const axisElementBuilders: Record<typeof selfBuilderAttrs[number], AxisElementBu
             tickModel: splitLineModel
         });
 
+        if (ticksCoords.length > 2 && (axis.scale as any).getInterval instanceof Function) {
```

Comment:
Create a new variable for `axis.scale as IntervalScale`

Reply:
`instanceof Function` can be omitted. If `getInterval` is not null, we can expect it to be a function.